ceph-dev-build/build/setup_pbuilder

```diff
@@ -83,6 +83,46 @@ echo "BUILD_HOME=`mktemp -d`" >> ~/.pbuilderrc
 # in newer versions. This ticket solves the specific issue in 8.1.1 (which vendors urllib3):
 # https://github.com/shazow/urllib3/issues/567
 echo "USENETWORK=yes" >> ~/.pbuilderrc
+# point gcc,g++ to the newly installed ones
+hookdir=$HOME/.pbuilder/hook.d
+if [ "$DIST" = "trusty" ]; then
+    cat >> ~/.pbuilderrc <<EOF
+OTHERMIRROR="deb http://ppa.launchpad.net/ubuntu-toolchain-r/test/ubuntu $DIST main"
+ALLOWUNTRUSTED=yes
+EXTRAPACKAGES="g++-7"
+HOOKDIR=$hookdir
+EOF
+fi
+
+if [ ! -e $hookdir ]; then
+    mkdir -p $hookdir
+    cat > $hookdir/E10update-gcc-alternatives <<EOF
```

Comment:
it looks overkill for using `update-alternatives` in a pbuilder chroot tarball. but actually, i was thinking about reusing this in `slave.yml` . so it's less intrusive when being applied to the jenkins slave node.